SpellGUIV2/App.xaml

```diff
@@ -11,6 +11,8 @@
                 <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/Blue.xaml" />
                 <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/BaseDark.xaml" />
                 <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.AnimatedSingleRowTabControl.xaml" />
+                <!--ResourceDictionary Source="pack://SiteOfOrigin:,,,/Languages/enUS.xaml"/-->
```

Comment:
I commented it out for debugging purposes. Using "SiteOfOrigin" is to read from the program directory.